CONTRIBUTING.md

```diff
@@ -28,15 +28,15 @@ from openadapt import models
 In order to effectively communicate any bugs or request new features, please select the appropriate form. If none of the options suit your needs, you can click on "Open a blank issue" located at the bottom.
 
 ## Testing
-[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behaviour and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
+[GitHub Actions](https://github.com/MLDSAI/OpenAdapt/actions/new) are automatically run on each pull request to ensure consistent behavior and style. The Actions are composed of PyTest, [black](https://github.com/psf/black) and [flake8](https://flake8.pycqa.org/en/latest/user/index.html).
 
-You can run these tests on your own computer by downloading the depencencies in requirements.txt and then running pytest in the root directory. 
+You can run these tests on your own computer by downloading the dependencies in requirements.txt and then running `pytest` in the root directory. 
```

Comment:
If we are getting rid of requirements.txt, you'll have to update this

Reply:
Great Point @Mustaballer,

@abrichr 
Should I remove requirements.txt in my PR which is updating the README ?
And then update README as well as remove the .venv setup ?

Reply:
Addressed in: [7e07cf4](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/7e07cf4b82c1f891d302727129b5d70018dc049c)